frontend/sentiment-dashboard/src/app/auth/auth.service.ts

```diff
@@ -0,0 +1,41 @@
+import { Injectable } from '@angular/core';
+import { Router } from '@angular/router';
+import { Observable } from 'rxjs';
+import { BehaviorSubject } from 'rxjs';
+import { from } from 'rxjs';
+import { map, tap, catchError } from 'rxjs/operators';
+import { of } from 'rxjs';
+import Amplify, { Auth } from 'aws-amplify';
+import { environment } from './../../environments/environment';
+
+
+@Injectable()
+export class AuthService {
+  public loggedIn: BehaviorSubject<boolean>;
+
+  constructor(private router: Router) {
+    Amplify.configure(environment.amplify);
+    this.loggedIn = new BehaviorSubject<boolean>(false);
+  }
+
+  public signIn(email, password): Observable<any> {
```

Comment:
Look into what type this returns - if possible, we should use a more specific type than 'any.'